FiveCalls/FiveCalls/AppDelegate.swift

```diff
@@ -39,7 +39,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window = UIWindow()
         if USE_NEW_SWIFTUI_INTERFACE {
             let store = Store(state: AppState(), middlewares: [appMiddleware()])
-            window?.rootViewController = UIHostingController(rootView: Dashboard().environmentObject(store))
+            let router = Router()
```

Comment:
see my comments elsewhere about 1) restricting this dashboard specific router to the dashboard and subviews and 2) hiding this behind the store so consumers don't need to know the internals about how routing works.